### PR TITLE
RUN-5679 BrowserView => View

### PR DESCRIPTION
**Pull request by pbaize**
#### Description of Change
Rename BrowserView to View. Leaves a private BrowserView to allow time to transition tests. Goes with this core pr https://github.com/HadoukenIO/core/pull/982

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [ ] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] Integration tests added in this repo and in the [test dashboard](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [x] relevant documentation is changed or added
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR release notes describe the change in a way relevant to app-developers
- [x] PR has assigned reviewers

#### Release Notes

Notes:

### CODE REVIEW

**Comment by aziz512** [comment]
build fail

**Comment by pbaize** [comment]
Successful build here: https://github.com/HadoukenIO/core/pull/982

**Review by MichaelMCoates — APPROVED** [review]
LGTM. Do we have to worry about merge ordering here?

**Comment by pbaize** [comment]
Wrong link in the PR

File: tutorials/View.getInfo.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Shows the View on a window.
+# Example
+```js
+let browserView;
+async function createView() {
```
Comment (licui3936) [inline tutorials/View.getInfo.md]: better change it to 'view'?

File: tutorials/View.getInfo.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Shows the View on a window.
+# Example
+```js
+let browserView;
```
Comment (licui3936) [inline tutorials/View.getInfo.md]: Better change it to 'view'?

File: tutorials/View.getInfo.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        target: me.identity, 
+        bounds: {x: 10, y: 10, width: 200, height: 200} 
+    });
+}
+
+async function getViewInfo() {
+    await createView();
+    console.log('View created.');
+    const me = await fin.Window.getCurrent();
+    browserView = fin.View.wrapSync({ uuid: me.identity.uuid, name: 'browserViewNameGetInfo' });
```
Comment (licui3936) [inline tutorials/View.getInfo.md]: same

File: src/api/fin.ts [hunk 4]
```diff
@@ -26,7 +26,8 @@ export default class Fin extends EventEmitter {
     public ExternalWindow: ExternalWindow;
     public Frame: _FrameModule;
     public GlobalHotkey: GlobalHotkey;
-    public BrowserView: BrowserViewModule;
+    public View: ViewModule;
+    private BrowserView: ViewModule;
```
Comment (licui3936) [inline src/api/fin.ts:30]: why do we still need 'BrowserView'? backward compatibility?

**Pull request closed without merging**